tests/test_event_intelligence.py

```
@@ -0,0 +1,316 @@
+        event_intelligence_dossier_from_dict(payload, require_digest=False)
+
+    payload = example()
+    payload.pop("dossier_sha256")
+    payload["knowledge_graph"]["nodes"][0]["relevance"]["estimate"] = float("inf")
+    with pytest.raises(EventIntelligenceValidationError, match="finite"):
+        event_intelligence_dossier_from_dict(payload, require_digest=False)
+
+    payload = example()
+    payload.pop("dossier_sha256")
+    payload["contextual_orchestrator"]["confidence"] = 1.1
+    with pytest.raises(EventIntelligenceValidationError, match="between 0.0 and 1.0"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)